packages/storybook/src/nimble/combobox/combobox.stories.ts

```diff
@@ -104,7 +103,7 @@ const metadata: Meta<ComboboxArgs> = {
             appearance="${x => x.appearance}"
             value="${x => x.currentValue}"
             placeholder="${x => x.placeholder}"
-            style="width: var(${menuMinWidth.cssCustomProperty});"
+            style="width: 250px;"
```

Comment:
menuMinWidth is defined as 176px, not 250, isn't this a change that was being avoided?

https://github.com/ni/nimble/blob/9a1ebfa1220710515a4fca079f8b01932fa3e347/packages/nimble-components/src/theme-provider/design-tokens.ts#L307-L309

Reply:
The instance of the `nimble-combobox` in Storybook specified a width of `176px`, but the component defined a `min-width:250px`, so the width was actually `250px` (and that works well for the Storybook preview). 

Since we're actually setting the min-width to `176px`, we're setting Storybook to `250px` so it's unchanged.

Reply:
Gotcha 👍